test/currency.json

```diff
@@ -7,5 +7,12 @@
   [1.99999, "$0.00", "$2.00"],
   [1.99, "$0.00", "$1.99"],
   [1.99, "$0", "$2"],
-  [1000.40, "$0", "$1000"]
+  [1000.40, "$0", "$1000"],
+  [500, "#,##0\"€\";(#,##0\"€\")", "500€"],
```

Comment:
I would also include the literal test from https://github.com/SheetJS/ssf/issues/19#issue-109139459

`[3.14, "#.00\"$\"", "3.14$"]`

Reply:
👍